src/containers/MovedResourcePage/MovedResourcePage.jsx

```diff
@@ -23,8 +23,11 @@ import { ResourceShape } from '../../shapes';
 const MovedResourcePage = ({ resource, locale, t }) => {
   const isLearningpath = !!resource.learningpath;
   const { data, loading, error } = useGraphQuery(searchQuery, {
-    variables: { 
-      ids: (isLearningpath ? resource.learningpath.id : resource.article.id).toString(),
+    variables: {
+      ids: (isLearningpath
+        ? resource.learningpath.id
+        : resource.article.id
+      ).toString(),
```

Comment:
Kunne forsåvidt ha vore skreve slik men din type er litt tydeligere.
```suggestion
      ids: (resource.learningpath?.id || resource.article.id).toString(),
```

Reply:
Tenkte på det. Gjorde det slik for at det skulle bli "symmetri" mellom ids og contextTypes. Vet ikke hva som er best

Reply:
Din løsning er bedre her.